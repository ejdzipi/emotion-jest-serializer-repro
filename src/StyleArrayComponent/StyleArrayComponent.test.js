import React from 'react'
import {mount} from 'enzyme'
import StyleArrayComponent from "./StyleArrayComponent";

describe('StyleArrayComponent', () => {
    it('enzyme - snapshot', () => {
        const component = mount(
            <StyleArrayComponent/>
        )

        expect(component).toMatchSnapshot()
    })
})
