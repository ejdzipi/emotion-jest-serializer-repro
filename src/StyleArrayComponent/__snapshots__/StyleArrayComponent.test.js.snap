// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyleArrayComponent enzyme - snapshot 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-1 {
  background-color: black;
  color: white;
}

<_default>
  <div
    className="emotion-0"
  >
    Test content
    <EmotionCssPropInternal
      __EMOTION_TYPE_PLEASE_DO_NOT_USE__="div"
      css={
        Array [
          Object {
            "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIlN0eWxlQXJyYXlDb21wb25lbnQuanN4Il0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQU9rQiIsImZpbGUiOiJTdHlsZUFycmF5Q29tcG9uZW50LmpzeCIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCdcclxuaW1wb3J0IHtjc3N9IGZyb20gJ0BlbW90aW9uL3JlYWN0J1xyXG5cclxuY29uc3Qgbm9BcnJheVN0eWxlID0gY3NzYFxyXG4gICAgZGlzcGxheTogZmxleDtcclxuYFxyXG5cclxuY29uc3Qgc3R5bGUxID0gY3NzYFxyXG4gICAgYmFja2dyb3VuZC1jb2xvcjogYmxhY2s7XHJcbmBcclxuXHJcbmNvbnN0IHN0eWxlMiA9IGNzc2BcclxuICAgIGNvbG9yOiB3aGl0ZTtcclxuYFxyXG5cclxuZXhwb3J0IGRlZmF1bHQgKCkgPT4ge1xyXG4gICAgcmV0dXJuIChcclxuICAgICAgICA8ZGl2IGNzcz17bm9BcnJheVN0eWxlfT5cclxuICAgICAgICAgICAgVGVzdCBjb250ZW50XHJcbiAgICAgICAgICAgIDxkaXYgY3NzPXtbc3R5bGUxLCBzdHlsZTJdfS8+XHJcbiAgICAgICAgPC9kaXY+XHJcbiAgICApO1xyXG59O1xyXG4iXX0= */",
            "name": "1ghjgev-style1",
            "styles": "background-color:black;label:style1;",
            "toString": [Function],
          },
          Object {
            "map": "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIlN0eWxlQXJyYXlDb21wb25lbnQuanN4Il0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQVdrQiIsImZpbGUiOiJTdHlsZUFycmF5Q29tcG9uZW50LmpzeCIsInNvdXJjZXNDb250ZW50IjpbImltcG9ydCBSZWFjdCBmcm9tICdyZWFjdCdcclxuaW1wb3J0IHtjc3N9IGZyb20gJ0BlbW90aW9uL3JlYWN0J1xyXG5cclxuY29uc3Qgbm9BcnJheVN0eWxlID0gY3NzYFxyXG4gICAgZGlzcGxheTogZmxleDtcclxuYFxyXG5cclxuY29uc3Qgc3R5bGUxID0gY3NzYFxyXG4gICAgYmFja2dyb3VuZC1jb2xvcjogYmxhY2s7XHJcbmBcclxuXHJcbmNvbnN0IHN0eWxlMiA9IGNzc2BcclxuICAgIGNvbG9yOiB3aGl0ZTtcclxuYFxyXG5cclxuZXhwb3J0IGRlZmF1bHQgKCkgPT4ge1xyXG4gICAgcmV0dXJuIChcclxuICAgICAgICA8ZGl2IGNzcz17bm9BcnJheVN0eWxlfT5cclxuICAgICAgICAgICAgVGVzdCBjb250ZW50XHJcbiAgICAgICAgICAgIDxkaXYgY3NzPXtbc3R5bGUxLCBzdHlsZTJdfS8+XHJcbiAgICAgICAgPC9kaXY+XHJcbiAgICApO1xyXG59O1xyXG4iXX0= */",
            "name": "pjmsmk-style2",
            "styles": "color:white;label:style2;",
            "toString": [Function],
          },
          "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIlN0eWxlQXJyYXlDb21wb25lbnQuanN4Il0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQW1CaUIiLCJmaWxlIjoiU3R5bGVBcnJheUNvbXBvbmVudC5qc3giLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnXHJcbmltcG9ydCB7Y3NzfSBmcm9tICdAZW1vdGlvbi9yZWFjdCdcclxuXHJcbmNvbnN0IG5vQXJyYXlTdHlsZSA9IGNzc2BcclxuICAgIGRpc3BsYXk6IGZsZXg7XHJcbmBcclxuXHJcbmNvbnN0IHN0eWxlMSA9IGNzc2BcclxuICAgIGJhY2tncm91bmQtY29sb3I6IGJsYWNrO1xyXG5gXHJcblxyXG5jb25zdCBzdHlsZTIgPSBjc3NgXHJcbiAgICBjb2xvcjogd2hpdGU7XHJcbmBcclxuXHJcbmV4cG9ydCBkZWZhdWx0ICgpID0+IHtcclxuICAgIHJldHVybiAoXHJcbiAgICAgICAgPGRpdiBjc3M9e25vQXJyYXlTdHlsZX0+XHJcbiAgICAgICAgICAgIFRlc3QgY29udGVudFxyXG4gICAgICAgICAgICA8ZGl2IGNzcz17W3N0eWxlMSwgc3R5bGUyXX0vPlxyXG4gICAgICAgIDwvZGl2PlxyXG4gICAgKTtcclxufTtcclxuIl19 */",
          "/*# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIlN0eWxlQXJyYXlDb21wb25lbnQuanN4Il0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQW1CaUIiLCJmaWxlIjoiU3R5bGVBcnJheUNvbXBvbmVudC5qc3giLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnXHJcbmltcG9ydCB7Y3NzfSBmcm9tICdAZW1vdGlvbi9yZWFjdCdcclxuXHJcbmNvbnN0IG5vQXJyYXlTdHlsZSA9IGNzc2BcclxuICAgIGRpc3BsYXk6IGZsZXg7XHJcbmBcclxuXHJcbmNvbnN0IHN0eWxlMSA9IGNzc2BcclxuICAgIGJhY2tncm91bmQtY29sb3I6IGJsYWNrO1xyXG5gXHJcblxyXG5jb25zdCBzdHlsZTIgPSBjc3NgXHJcbiAgICBjb2xvcjogd2hpdGU7XHJcbmBcclxuXHJcbmV4cG9ydCBkZWZhdWx0ICgpID0+IHtcclxuICAgIHJldHVybiAoXHJcbiAgICAgICAgPGRpdiBjc3M9e25vQXJyYXlTdHlsZX0+XHJcbiAgICAgICAgICAgIFRlc3QgY29udGVudFxyXG4gICAgICAgICAgICA8ZGl2IGNzcz17W3N0eWxlMSwgc3R5bGUyXX0vPlxyXG4gICAgICAgIDwvZGl2PlxyXG4gICAgKTtcclxufTtcclxuIl19 */",
        ]
      }
    >
      <div
        className="emotion-1"
      />
    </EmotionCssPropInternal>
  </div>
</_default>
`;
